lib/vanagon/utilities.rb

```diff
@@ -281,7 +280,7 @@ def local_command(command, return_command_output: false)
       clean_environment do
         puts "Executing '#{command}' locally"
         if return_command_output
-          ret = %x(#{command}).chomp
+          ret = `#{command}`.chomp
```

Comment:
I kind of prefer `%x()`; [we can tune that](https://github.com/bbatsov/rubocop/blob/master/config/default.yml) using this in rubocop.yaml:

```yaml
Style/CommandLiteral:
  EnforcedStyle: percent_x
```